robot/web/stream/state/state.go

```diff
@@ -422,8 +424,10 @@ func (ss *StreamState) restart(ctx context.Context) {
 }
 
 func (ss *StreamState) streamH264Passthrough(ctx context.Context) error {
-	cam, err := camera.FromRobot(ss.robot, ss.Stream.Name())
+	shortName := resource.SDPTrackNameToShortName(ss.Stream.Name())
```

Comment:
We need to turn the SDPTrackName back into a short name for the FromRobot lookup to work if the camera is in a remote part

Reply:
If the camera is not in a remote part then there is no behavior change